Dockerfile

```diff
@@ -75,30 +71,41 @@ ENV DJANGO_SETTINGS_MODULE cfgov.settings.production
 ENV DJANGO_STATIC_ROOT ${STATIC_PATH}
 ENV ALLOWED_HOSTS '["*"]'
 
+# Install and enable SCL-based Apache server and mod_wsgi,
+# and converts all Docker Secrets into environment variables.
 RUN yum -y install ${SCL_HTTPD_VERSION} ${SCL_PYTHON_VERSION}-mod_wsgi && \
     yum clean all && rm -rf /var/cache/yum && \
-    echo "source scl_source enable ${SCL_HTTPD_VERSION}" > /etc/profile.d/enable_scl_httpd.sh
+    echo "source scl_source enable ${SCL_HTTPD_VERSION}" > /etc/profile.d/enable_scl_httpd.sh && \
+    echo '[ -d /var/run/secrets ] && cd /var/run/secrets && for s in *; do export $s=$(cat $s); done && cd -' > /etc/profile.d/secrets_env.sh
 
 # See .dockerignore for details on which files are included
-COPY . .
+COPY --chown=apache:apache . .
+
+RUN yum -y install nodejs yarn  && \
+    yum clean all && rm -rf /var/cache/yum && \
+    chown -R apache:apache ${APP_HOME} ${SCL_HTTPD_ROOT}/usr/share/httpd ${SCL_HTTPD_ROOT}/var/run
+
+# Remove files flagged by image vulnerability scanner
+# FIXME: Find out if these packages can be remove altogether
+RUN cd /opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/ && \
+    rm -f ndg/httpsclient/test/pki/localhost.key sslserver/certs/development.key
```

Comment:
So, `django-sslserver` probably doesn't need to be installed, as it is only used to add a `runserver` analog `runsslserver` for testing with SSL locally. But it [comes from `local.txt` above](https://github.com/cfpb/cfgov-refresh/pull/5553/files#diff-3254677a7917c6c01f55212f86c57fbfR43), and I'm not sure how we might want to ensure it doesn't get this far. We want to retain it for local development usage.

ndg-httpsclient is a requirement for our Akamai edgegrid-python package and we can't really do much about it, I don't think.

Reply:
@willbarton, thanks for checking into this.  I don't think we need to change anything now.  I'll just take out the `FIXME` comment for now.  Over time if we start having a bunch of one-off _fixes_ to make the vulnerability scanner happy, we can adjust, perhaps tuning/whitelisting on the scanner instead.